monzo/monzo.py

```diff
@@ -276,3 +277,57 @@ def update_transaction_notes(self, transaction_id, notes):
            :rtype: The updated transaction object.
         """
         return self.update_transaction_metadata(transaction_id, 'notes', notes)
+
+    def create_receipt(self, transaction_id, total, currency, items, taxes=None, payments=None, merchant=None):
```

Comment:
I'm thinking it might be worth requiring the receipts to be generated beforehand and then just passed to this method.
We can provide some helper functions elsewhere to combine the receipts into one but I think this method should just take a single receipt dict and an external_id.

Reply:
Yes this would be much simpler. However, this would mean that params `taxes, payments & merchant` would no longer be required.

Reply:
I think use the case where the receipt is pregenerated. With the relative complexity of receipts and the scope for reuse it makes more sense imo.